napi-inl.h

```diff
@@ -2523,8 +2523,8 @@ inline ClassPropertyDescriptor<T> ObjectWrap<T>::InstanceAccessor(
 
   napi_property_descriptor desc = {};
   desc.utf8name = utf8name;
-  desc.getter = T::InstanceGetterCallbackWrapper;
-  desc.setter = T::InstanceSetterCallbackWrapper;
+  desc.getter = getter ? T::InstanceGetterCallbackWrapper : nullptr;
+  desc.setter = setter ? T::InstanceSetterCallbackWrapper : nullptr;
```

Comment:
Just fyi, I think we have a slight preference for using explicit nullptr comparisons (i.e. `getter != nullptr ? …`). You can update the PR if you want but this should be fine either way.

Reply:
Thank you for review.
I addressed your comment :)